src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -96,19 +104,21 @@ public void updateGameState(GameStatus gameStatus, String gameId){
     }
 
     public Player getPlayer(String gameCode, String username){
-        return getGameByCode(gameCode).getPlayers().stream().filter(p -> p.getPlayerUsername().equals(username)).findFirst().get();
+        return getGameByCode(gameCode)
+                .getPlayers()
+                .stream()
+                .filter(p -> p.getPlayerUsername()
+                        .equals(username))
+                .findFirst()
+                .get();
     }
 
     public ArrayList<Player> getLeaderboard(String gameId){
         ArrayList<Player> players = getGameByCode(gameId).getPlayers();
          Collections.sort(players, new Comparator<Player>() {
             @Override
             public int compare(Player o1, Player o2) {
-                if(o1.getTotalPoints() > o2.getTotalPoints()){
-                    return -1;
-                }else {
-                    return 1;
-                }
+                return (o1.getTotalPoints() > o2.getTotalPoints()) ? -1 : 1;
```

Comment:
refactored if/else statement to return a value based on simple conditional statement with lambda to improve readability and reduce lines of code.
This is recommend by the "Ternary Operator" section of the java coding best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3